### PR TITLE
Optimized chunk loading

### PR DESCRIPTION
**Pull request by jonsherrard**
A couple o' pesky bugs confusing things.

### CODE REVIEW

File: src/config/createWebpackConfig.js [hunk 1]
```diff
@@ -203,12 +204,26 @@ module.exports = (target = 'node', options) => {
...
+            chunks: 'all'
+          }
+        },
         chunks: 'all',
         name: false
       }
     }
   }
+  config.plugins.push(
+    new CleanPlugin(['.tapestry'], {
```
Comment (thomasdigby) [inline src/config/createWebpackConfig.js]: Will this clean it for both `server` and `client`? Does that matter? When I thought about this earlier I assumed the server would fill `.tapestry` then the client webpack compiler would clean it again before adding the client files.

Comment (jonsherrard) [reply src/config/createWebpackConfig.js]: It seems that it only cleans the files that's webpack's `output: {}` -ing at that time. Or I've got something weird going with my setup and it's not actually working...

File: src/server/render/default-document/index.js [hunk 2]
```diff
@@ -21,7 +22,9 @@ const escapeScriptTags = data => {
 const getProductionBundle = () => {
   const assetsPath = path.resolve(process.cwd(), '.tapestry', 'assets.json')
   const assets = fs.readJsonSync(assetsPath)
-  return assets.client.js
+  return map(assets, asset => (
```
Comment (thomasdigby) [inline src/server/render/default-document/index.js]: Hatin on lodash atm. Could we use `Object.values()` here? `return Object.values(assets).map(({ js })=> '<script src={js} />)`

File: src/config/createWebpackConfig.js (hunk 1 above)
Comment (thomasdigby) [reply src/config/createWebpackConfig.js]: oh cool, that's pretty spesh.

File: src/server/render/default-document/index.js (hunk 2 above)
Comment (jonsherrard) [reply src/server/render/default-document/index.js]: Now that's what I call ES2015

File: src/server/render/default-document/index.js [hunk 3]
```diff
@@ -22,9 +21,7 @@ const escapeScriptTags = data => {
 const getProductionBundle = () => {
   const assetsPath = path.resolve(process.cwd(), '.tapestry', 'assets.json')
   const assets = fs.readJsonSync(assetsPath)
-  return map(assets, asset => (
-    <script src={asset.js} />
-  ))
+  return Object.values(assets).map(({ js })=> (<script src={js} />))
```
Comment (thomasdigby) [inline src/server/render/default-document/index.js]: that was totally pseudo code. _it works??_

Comment (jonsherrard) [reply src/server/render/default-document/index.js]: yep just added implicit JSX return 👌

**Pull request merged**